File: pmd/java/listas/listaex01/ex2/ContaInterface.java
package listas.listaex01.ex2;

import java.util.Scanner;

public class ContaInterface {

  public static void menu(Conta conta, Scanner scn) {

    while (true) {

      System.out.print(
          "\n0 - SAIR" +
              "\n1 - DEPOSITAR" +
              "\n2 - SACAR" +
              "\n3 - VER SALDO" +
              "\nINFORME A SUA OPÇÃO: ");

      int op = scn.nextInt();

      switch (op) {

        case 0:
          return;

        case 1:
          System.out.print("\nINFORME O VALOR DO DEPOSITO: ");
          conta.setSaldo(conta.getSaldo() + scn.nextDouble());
          break;

        case 2:
          System.out.print("\nINFORME O VALOR DO SAQUE: ");
          conta.setSaldo(conta.getSaldo() - scn.nextDouble());
          break;

        case 3:
          System.out.println("\nSALDO ATUAL: " + conta.getSaldo());
      }
    }
  }

}

File: pmd/python/listas/listaex01/ex2/containterface.py
from conta import Conta

class ContaInterface:

    @staticmethod
    def executar(conta):
        while True:
            menu =  '\n\n0 - Sair\n1 - Fazer deposito' + \
                    '\n2 - Sacar valor\n3 - Conferir saldo' + \
                    '\nDigite sua opcao: '

            op = int(input(menu))

            #python não tem estrutura condicional de caminho múltiplo (como o switch do C)
            if op == 0:
                return
            
            elif op == 1:
                conta.depositar(float(input('Informe o valor: ')))
                print('\nDeposito efetuado.\nSaldo atual: ', conta.saldo()) 
                
            elif op == 2:
                if conta.sacar(float(input('Informe o valor: '))):
                    print('\nSaque efetuado.\nSaldo atual: ', conta.saldo())
                else:
                    print('\nSaque nao efetuado.\nSaldo insuficiente.') 
                
            elif op == 3:
                print('\nSaldo atual: ', conta.saldo())
            
            else:
                print('\nOpcao invalida.')
